string smer
int16 hitrost
int16 trajanje
